net.brinkervii.grapejuice.yml

```diff
@@ -127,15 +136,24 @@ modules:
       - mv -v /app/share/applications/roblox-studio.desktop /app/share/applications/${FLATPAK_ID}.robloxstudio.desktop
       - mv -v /app/share/applications/roblox-studio-auth.desktop /app/share/applications/${FLATPAK_ID}.robloxstudioauth.desktop
       # Edit Grapejuice shortcut
-      - desktop-file-edit /app/share/applications/${FLATPAK_ID}.desktop --set-icon=${FLATPAK_ID} --set-key=Exec --set-value='grapejuice gui'
```

Comment:
Just a heads up: These are harmless formatting changes, introduced after running [flatpak-external-data-checker](https://github.com/flathub/flatpak-external-data-checker) with the `--update` flag (which is what the Flathub bot does). The desktop entries are still valid.